gains_workshops.rst

```diff
@@ -0,0 +1,24 @@
+GAINS Workshops
+^^^^^^^^^^^^^^^^^^
+
+The aim of the workshop is to help new users of the GAINS
+modelling framework getting started with their modelling work.
+The main features of the “framework” are introduced, and the
+use cases of some features are shown. The user can learn how
+to build their own emission scenarios, adjust and create new policy
+constraints and run analysis on costs and co - benefit.
+For information about the model, its structure, mathematical
+formulation and much more, please see the `documentation <http://gains.iiasa.ac.at/models/gains_resources.html>`_.
+
+.. contents:: Table of Content
+   :local:
+
+GAINS Capacity building
```

Comment:
Not quite what this line does? Is this meant as a heading? If yes, please add syntax highlighting